templates/zerver/help/mute-a-stream.md

```diff
@@ -0,0 +1,23 @@
+### Mute a stream
+
+Muting streams to reduce notification clutter is easy. Simply hover your mouse
+over the stream in the streams list on the left of the chat, press the down
+chevron button
```

Comment:
OK, there's multiple issues here. First, the image isn't showing up. Second, we have multiple pictures of the down chevron button, but most of our docs use the `![chevron](static/images/help/chevron.png)`. So, to fix this, just replace your line with `press the chevron (![chevron](static/images/help/chevron.png)) icon`.